docs/beaver-builder/developer/custom-modules/cmdg-15-helpers.md

```diff
@@ -12,32 +12,41 @@ The CSS helper makes it simple to render complex CSS rules with your settings,
 especially settings for multiple device sizes. Just use the methods described
 below in your `frontend.css.php` and we'll handle the rest for you.
 
+### Render a single rule for a single device size.
```

Comment:
Super picky but remove period at end of this line because it's a heading. Also super nitpicky, headings sometimes use "Render" and sometimes "Renders" throughout this article. Either is ok but change to be consistent?